spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -1066,44 +1087,57 @@ public void setMaxHttpFormPostSize(DataSize maxHttpFormPostSize) {
 			this.maxHttpFormPostSize = maxHttpFormPostSize;
 		}
 
+		@Deprecated
+		@DeprecatedConfigurationProperty(replacement = "server.jetty.acceptors")
```

Comment:
That replacement value is wrong.

Reply:
Good catch. Updated.

@snicoll Boot MR question... Should I rebase and squash subsequent commits to MR based on feedback (like this case) or just add more commits and let Github squash at end of MR?

Reply:
Not a single answer here. When it's about fixing a PR review (as you did), we prefer a single squashed commit rebased on top of the latest state of the target branch (here `master`). When there is a discussion and you'd like to suggest something, a separate commit is better in case we don't pursue with the suggestion.